layer/training/runtime/system_metrics.py

```diff
@@ -0,0 +1,227 @@
+import subprocess  # nosec: import_subprocess
+import time
+from logging import Logger
+from time import sleep
+from typing import Any, Dict, Tuple
+from uuid import UUID
+
+import nvsmi  # type: ignore
+import polling  # type: ignore
+import psutil  # type: ignore
+
+from layer.clients.layer import LayerClient
+from layer.logged_data.log_data_runner import LogDataRunner
+
+
+class SystemMetrics:
+    cpu_used_temp = None
+    start_time_temp = None
+    step_value = 0
```

Comment:
You can define these inside the constructor, since different SystemMetrics instances would share the same values, which is not desirable: https://dzone.com/articles/python-class-attributes-vs-instance-attributes